### DISCUSSION TITLE
SQLite for production

### DISCUSSION THREAD

**Discussion by johnnypea** [ueberdosis/hocuspocus#533] [Q&A] [answered]
Why you don't recommend using SQLite in production?

I can you stated in this MR https://github.com/ueberdosis/hocuspocus/pull/267 :

> It’s not yet documented because the extension stores every single update, which can kill your production environment. We need to add support for debouncing and cleaning up the history. So if you see it here: Feel free to fiddle around with it, but don’t use it in production (yet).

Is this still the case? Wasn't this resolved?

And then after server starts there is this warning:

> The SQLite extension is intended to be used in a local development environment, not in a production environment.

Thank you!

**Comment by janthurau** [comment, marked as answer]
hey @johnnypea, there's no particular reason, just in general I wouldn't use SQLite in prod. Probably it's fine for you though (https://www.sqlite.org/whentouse.html)

**Reply by johnnypea** [reply]
I was just curious as I have great experience using it in production for small to medium projects. Straightforward implementation, can accommodate quite complex data and it can withstand enormous load if setup well.

I don't think it is reasonable to _discourage_ people using it (always reminding them in CLI) without any particular reason. 

Thanks for your answer though.

**Reply by janthurau** [reply]
yep, I think that notice is quite old. I'll remove it - thx!
